crates/cli/src/commands.rs

```diff
@@ -60,3 +63,12 @@ impl Default for Format {
         }
     }
 }
+
+impl Display for Format {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Format::Human => f.write_str("human"),
+            Format::GithubAnnotationNative => f.write_str("github-annotation-native"),
+        }
+    }
+}
```

Comment:
nit: I wonder if we ca generate these automatically with `strum` and something like below?

```
use strum_macros::Display;

#[derive(Display)]
#[strum(serialize_all = "kebab-case")]
enum Status {
    InProgress,
    NotStarted,
    UnderReview,
}
```

Reply:
Thank you for interesting to merge this PR, and of course this great library.

> That nit which I think is a nice little win to avoid typos

I usually avoid using library (crate) for small thing like this because:
1. you need to maintain the crate version
2. you need to go to that crate's repository to read code when you want to know what it does inside. feels it's overkill for what I want to do.

But you don't hesitate to add dependency to library, then I will add it and rewrite as you suggested (I also think that is clean tbh, I just worried to adding dependency).

> I think it might be worth writing a ui-test for this as it's already once slipped through our checks.

I would love to add test for it! but it might take some time like couple days, so if you want to merge this PR asap then 
 I am sorry but please consider adding it yourself.

Reply:
We’re already using this crate (just not in the one we’re currently working on), so it would be great to make it a workspace dependency.

Reply:
> I usually avoid using library (crate) for small thing like this ...

@tomocrafter, amen to that!

As @gvozdvmozgu pointed out, we already use it pretty heavily like [here](https://github.com/quarylabs/sqruff/blob/fb6adeb5b9544b76415bcbf949240882cb889396/crates/lib-core/src/dialects/syntax.rs#L9) so that's the only reason. 

I'll make the change, write the tests, and tack it on to this branch! Thanks again @tomocrafter!

@gvozdvmozgu good point on also making this a workspace dependency.

Reply:
Oh I completely missed that, then there is no reason to use it!
Thank you for the support @benfdking adnd @gvozdvmozgu !